src/main/java/br/ufpb/dcx/esa/medievalbank/service/BurgosAgency.java

```diff
@@ -3,24 +3,24 @@
 
 public class BurgosAgency {
 
+	private static String name;
+	private static String manager;
+
+
 	public static void setName(String string) {
-		// TODO Auto-generated method stub
-		
+		BurgosAgency.name = string;
```

Comment:
Rename string to name
```suggestion
	public static void setName(String name) {
		BurgosAgency.name = name;
```